cargo-dist/tests/snapshots/install_path_no_fallback_taken.snap

```diff
@@ -89,8 +86,8 @@ The installer for axolotlsay 0.2.1
 
 This script detects what platform you're on and fetches an appropriate archive from
 https://github.com/axodotdev/axolotlsay/releases/download/v0.2.1
-then unpacks the binaries and installs them to
-$INSTALL_PATH_HINT
+then unpacks the binaries and installs them to one of the following locations
+\$HOME/.axolotlsay\$MY_ENV_VAR/My Axolotlsay Documents/bin
```

Comment:
looks like missing a newline

Reply:
Ugh yes, I get the jinja whitespace squelching stuff wrong at least 50% of the time